service/src/main/resources/application.yml

```diff
@@ -69,6 +69,14 @@ management:
       exposure:
         # Expose all management endpoints in Spring Boot
         include: "*"
+  metrics:
+    distribution:
+      # Used to publish a histogram suitable for computing aggregable (across dimensions) percentile
+      # latency approximations in Prometheus (by using histogram_quantile)
+      # For more information: https://micrometer.io/docs/concepts#_histograms_and_percentiles
+      minimum-expected-value[http.server.requests]: 200ms
+      maximum-expected-value[http.server.requests]: 60s
+      percentiles-histogram[http.server.requests]: true
```

Comment:
These configurations tell Micrometer to use its default buckets falling between these expected values.  Setting our expectations keeps us from gathering metrics that we know won't be meaningful (buckets that are always empty).

Even so, that's a lot of metrics, looks like ~40 for each observed combo of endpoint, status, exception, etc!  We could narrow down our bucket selection using the `slo` property instead, but I'm inclined to stick with the defaults to start.

From https://engineering.statefarm.com/blog/observing-latency-tail/ --
> Picking buckets to configure can be difficult. The buckets are used to create an estimate, so more buckets gives you a more accurate estimate but each one creates another time series. In Prometheus each time series is a datapoint to be tracked and stored. For most applications I pick 5-10 buckets as a starting point.
>
> Selecting the values for buckets I start just below your target latency, increasing to 4x the target latency. This is just a starting point and will be adjusted as needed for each application.
>
> For example, the configuration for a service with a target latency of 100ms:
>
>```
>management.metrics:
>  distribution:
>    slo[http.server.requests]: "75ms, 100ms, 150ms, 200ms, 250ms, 300ms, 350ms, 400ms"
>```